release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -479,6 +479,14 @@ OpenShift Kubernetes Engine in order to better communicate what value the
 product offering delivers. For more information, see
 link:https://docs.openshift.com/container-platform/4.4/welcome/oke_about.html[About OpenShift Kubernetes Engine].
 
+[id="ocp-4-4-aro-4x-docs-available"]
+==== Documentation is now available for the 4.3 version Azure Red Hat OpenShift.
+
+This new version is jointly managed, supported, and documented by both Red Hat and Microsoft
```

Comment:
We should probably add some form of punctuation at the end of this sentence. A colon probably makes the most sense here.

Reply:
Woops. I meant to remove that from the heading and missed.